com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
-### Refraction model
+<a name="Refraction-models"></a>
 
-In the Lit family of materials, when the surface type is set to *Transparent*, you can select between *None*, *Box*, *Sphere* or *Thin* refraction models.
+### Refraction models
 
-For path tracing, the distinction between *Box* or *Sphere* makes no sense (as rays can intersect the real objects in the scene), and both effectively carry the common meaning of a *thick* mode, to be used on solid objects represented by a closed surface. On the other hand, *Thin* conveys the same idea as its rasterized version, and *None* is a special case of thin refractive surface, hardcoded to be fully smooth to simulate alpha blending. Additionally, transparent surfaces should be *Double-Sided*, so that they get intersected from both sides, and normal mode should be selected appropriately for each situation, as described right below.
+Path tracing changes the way refraction models behave. The table below describes how each refraction model behaves when you enable path tracing. To change the type of refraction model a Lit Material uses, set its **Surface type** to **Transparent** then select a method from the **Refraction model** drop down.
 
-| Refraction model  | Path tracing meaning                              | Surface sidedness                                 |
-|-------------------|---------------------------------------------------|---------------------------------------------------|
-| *Box* or *Sphere* | *Thick* object (e.g magnifying paperweight)       | Double sided, with *None* normal mode             |
-| *Thin*            | *Thin* object (e.g soap bubble or window)         | Double sided, with *Flip* or *Mirror* normal mode |
-| *None*            | *Thin* object, with smoothness = 1 and no Fresnel | Double sided, with *Flip* or *Mirror* normal mode |
+| Refraction model | Path tracing behavior                                        | Compatible Surface sides                                     |
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Box**          | A box-shaped model where incident light enters through a flat surface and leaves through a flat surface.<br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Sphere**       | A sphere-shaped model that produces a magnifying glass-like effect to refraction. <br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
```

Comment:
Box and Sphere are **exactly** the same in path tracing, they correspond to a **thick** mode. It's a bit confusing to have them separate in this table, as if they have different effects.